src/openms/include/OpenMS/ANALYSIS/ID/IDRipper.h

```diff
@@ -70,10 +70,14 @@ namespace OpenMS
       @param proteins Peptide identification annotated with file origin
       @param peptides Protein identification
     */
-    void rip(std::map<String, std::pair<std::vector<ProteinIdentification>, std::vector<PeptideIdentification> > > & ripped, std::vector<ProteinIdentification> & proteins, std::vector<PeptideIdentification> & peptides);
+    void rip(std::map<std::pair<UInt, UInt>, std::pair<std::vector<ProteinIdentification>, std::vector<PeptideIdentification> > > & ripped, std::vector<ProteinIdentification> & proteins, std::vector<PeptideIdentification> & peptides);
```

Comment:
this type looks very ugly.
Can you refactor that into a struct? or a class with members?

Reply:
could make pyOpenMS changes (see pxd file) necessary